Web Quiz Engine (Kotlin)/task/src/engine/entity/QuizResponseBody.kt

```diff
@@ -0,0 +1,7 @@
+package engine.entity
+
+sealed class QuizPostResponseBody(val success: Boolean, val feedback: String)
```

Comment:
Sould all me in a `model` package

Reply:
I sould you all you want though I did put all the content of the `entity` package into the `model` one and then deleted it. Done